### ISSUE TITLE
Feature Request: .NET C#

### ISSUE THREAD

**Issue by PixelBrewer** [bootdotdev/curriculum#27]
I have found that a very common backend framework in enterprise applications is modern .NET and I feel like if this course is focused on backend development it may be advantageous to introduce students to a tech stack that is a bit older than Go or Rust. It would also be a great learning opportunity for students to understand older language architectures and how OOP focused languages like C# or Java work under the hood. It could also provide experience with some alternative points of view for similar tasks mentioned in the Go course. Much of the Go course is oriented around web servers and the same can be achieved in .NET fairly easily.

**Comment by famasf1** [comment]
Agreed. A lot of SEA-based job are also using C# more than Go or even Python (in fact, many companies in my country (Thailand) considered Go as cutting-edge. So many companies basically doesn't want to adopted them and preferred to stick with C# shockingly enough.)

I think this will definitely helped tons of student that stay in an country where Go or Python is not as popular as C#.
